File: src/main/java/ru/mdashlw/hypixel/reply/impl/KeyReply.java
package ru.mdashlw.hypixel.reply.impl;

import com.fasterxml.jackson.annotation.JsonProperty;
import ru.mdashlw.hypixel.entities.Key;
import ru.mdashlw.hypixel.reply.Reply;

public class KeyReply extends Reply<Key> {
    @JsonProperty
    private Key record;

    @Override
    public Key getValue() {
        return record;
    }
}

File: src/main/kotlin/ru/mdashlw/hypixel/api/reply/impl/KeyReply.kt
package ru.mdashlw.hypixel.api.reply.impl

import ru.mdashlw.hypixel.api.entities.Key
import ru.mdashlw.hypixel.api.reply.Reply

class KeyReply(
    success: Boolean,
    cause: String?,
    throttle: Boolean,
    record: Key?
) : Reply<Key>(success, cause, throttle, record)
